fastapi_asyncapi/asyncapi.py

```diff
@@ -80,18 +132,21 @@ def get_asyncapi(
             tags=tags,
             servers=servers,
             channels=channels,
+            components=components,
         ),
         by_alias=True,
         exclude_none=True,
     )
 
 
 def get_asyncapi_html(
-    *,
-    asyncapi_url: AnyHttpUrl,
-    title: str,
-    asyncapi_js_url: str = "https://unpkg.com/@asyncapi/web-component@1.0.0-next.32/lib/asyncapi-web-component.js",  # noqa: E501
-    asyncapi_css_url: str = "https://unpkg.com/@asyncapi/react-component@1.0.0-next.32/styles/default.min.css",  # noqa: E501
+        *,
+        asyncapi_url: AnyHttpUrl,
+        title: str,
+        asyncapi_js_url: str = "https://unpkg.com/@asyncapi/react-component@latest/browser/standalone/index.js",
+        # noqa: E501
+        asyncapi_css_url: str = "https://unpkg.com/@asyncapi/react-component@latest/styles/default.min.css",
+        # noqa: E501
```

Comment:
Why changing the indentation?

Reply:
I think that PyCharm did that automatically. But i can change that if you like